app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  def recent_posts
+    posts.order(created_at: :desc).limit(3)
+  end
+end
```

Comment:
- [ ] Kindly add associations between user and `likes, comments, posts `models.

Reply:
Addressed